Sources/tart/DiskImageFormat.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import ArgumentParser
+
+enum DiskImageFormat: String, CaseIterable, Codable {
+  case raw = "raw"
+  case asif = "asif"
+
+  var displayName: String {
+    switch self {
+    case .raw:
+      return "RAW (UDIF read-write)"
+    case .asif:
+      return "ASIF (Apple Sparse Image Format)"
+    }
+  }
+
+  var description: String {
+    switch self {
+    case .raw:
+      return "Traditional disk image format, compatible with all macOS versions"
+    case .asif:
+      return "High-performance sparse disk image format, requires macOS 15+ (creation requires macOS 26+)"
```

Comment:
```suggestion
      return "High-performance sparse disk image format, requires macOS 26 or newer"
```

Also, not sure if `description` is even used at all (dead code), we can just probably remove it altogether.